### PR TITLE
changes to add start_time and end_time filters to GetAlertsRequest

### PR DESCRIPTION
**Pull request by sbcd90**
### Description
changes to add start_time and end_time filters to GetAlertsRequest
 
### Issues Resolved
[List any issues this PR will resolve]
 
### Check List
- [X] New functionality includes testing.
  - [X] All tests pass
- [ ] New functionality has been documented.
  - [ ] New functionality has javadoc added
- [X] Commits are signed per the DCO using --signoff 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

**Review by riysaxen-amzn — COMMENTED** [review]
can we link this issue to the PR -> https://github.com/opensearch-project/security-analytics/issues/797

File: src/main/java/org/opensearch/securityanalytics/alerts/AlertsService.java [hunk 1]
```diff
@@ -305,4 +324,17 @@ public void onFailure(Exception e) {
         }
 
     }
+
+    private static BoolQueryBuilder getBoolQueryBuilder(Instant startTime, Instant endTime) {
```
Comment (riysaxen-amzn) [inline src/main/java/org/opensearch/securityanalytics/alerts/AlertsService.java:328]: we have a similar method in FindingsService, we can extract this out in helper class

**Review by riysaxen-amzn — APPROVED** [review]

File: src/main/java/org/opensearch/securityanalytics/alerts/AlertsService.java (hunk 1 above)
Comment (sbcd90) [reply src/main/java/org/opensearch/securityanalytics/alerts/AlertsService.java:328]: this will taken up in a future pr as part of #797

**Review by amsiglan — APPROVED** [review]

File: src/main/java/org/opensearch/securityanalytics/alerts/AlertsService.java (hunk 1 above)
Comment (amsiglan) [reply src/main/java/org/opensearch/securityanalytics/alerts/AlertsService.java:328]: Let's not close this issue then

**Pull request merged**